### DISCUSSION TITLE
[MacOS] Couldn't load shared library

### DISCUSSION THREAD

**Discussion by Tempomancy** [wcko87/beatoraja-english-guide#45] [beatoraja questions / troubleshooting]
I'm trying to launch beatoraja on Mac when I press play on the config window, most of the tabs turn grey and in my terminal I get an error
`Couldn't load shared library`

<img width="657" alt="uh" src="https://github.com/wcko87/beatoraja-english-guide/assets/160374546/e5a90501-9362-4450-9082-396527bfba34">

Anyone know what I could do to fix this?

**Comment by wcko87** [comment]
We resolved this issue over discord.
The issue was due to downloading the ARM version of Liberica instead of x86 Liberica.

The correct version of Liberica Java to download is x86 - 64bit - Full.
It is the same issue as this post: https://github.com/wcko87/beatoraja-english-guide/discussions/41
